\mysection{8}{\color{eubicRed} Poster Flash Talks of Young Investigators}

\noindent\textbf{Day 2 -- 11. January 2017}

\begin{table}[!h]
  \centering
  \begin{tabular}{ | L{0.23\textwidth} | L{0.7\textwidth} | }
    \hline
    Ludger Goeminne       & MSqRob: analysis of label-free proteomics data in an R/Shiny environment                                                                                                  \\
    \hline
    %Stefan Senn           & Automated Annotation of Masses of Highly Modified Proteins with ModFinder                                                                                                 \\
    %\hline
    Johannes Griss        & The spectra-cluster toolsuite: Enhancing proteomics analysis through spectrum clustering                                                                                  \\
    \hline
    Emma Ricart Altimiras & A Bioinformatics Tool for Nonribosomal Peptides Identification by Tandem Mass Spectrometry                                                                                \\
    \hline
    Sebastian Dorl        & Identifying tandem mass spectra of phosphorylated peptides before database search using machine-learning                                                                  \\
    \hline
    David Hollenstein     & MasPy – mass spectrometry-based proteomics data analysis with python                                                                                                      \\
    \hline
    Adriaan Sticker       & Mass spectrometrists should search for all peptides, but assess only the ones they care about                                                                             \\
    \hline
    Surya Gupta           & An unbiased protein association study on the public human proteome reveals biological connections between co-occurring protein pairs                                      \\
    \hline
    Christophe Bruley     & Proline: a software environment for label-free quantification data analysis and exploration                                                                               \\
    \hline
  \end{tabular}
\end{table}

\noindent\textbf{Day 3 -- 12. January 2017}

\begin{table}[!h]
  \centering
  \begin{tabular}{ | L{0.23\textwidth} | L{0.7\textwidth} | }
    \hline
    Hugo López-Fernández  & Mass-Up and Decision Peptide-Driven: two open-source applications for MALDI-TOF MS data analysis and protein quantification                                               \\
    \hline
    Thilo Muth            & Analyzing metaproteome samples on the go: the full-featured MPA portable software provides protein identification enriched with taxonomic and functional meta-information \\
    \hline
    Corinna Klein         & Electronic sample management and archiving system for proteomics MS-data                                                                                                  \\
    \hline
    Adithi Varadarajan    & An integrated proteogenomics approach to discover the entire protein-coding potential of prokaryotic genomes                                                              \\
    \hline
    Roman Mylonas         & MsViz, a zero learning curve graphical software tool for detailed manual validation and quantitation of post-translational modifications                                  \\
    \hline
    Dominik Kopczynski    & PeptideMapper: Efficient and Versatile Amino Acid Sequence and Tag Mapping                                                                                                \\
    \hline
    Felix Van der Jeugt   & Unipept: Tryptic Peptide-Based Biodiversity Analysis of Metaproteome Samples                                                                                              \\
    \hline
  \end{tabular}
\end{table}
